### PR TITLE
Episode 16.1 - Edda Half Moon In The Daylight

### PR DESCRIPTION
**Pull request by Everade**
* **Addressed Issue(s)**: 
https://github.com/rathena/rathena/issues/6450

* **Server Mode**: 
Renewal

* **Description of Pull Request**: 
Implementation of the Episode 16.1 - Edda Half Moon In The Daylight - Memorial Dungeon.

### CODE REVIEW

File: npc/re/merchants/enchan_edda_half_moon.txt [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+		specialeffect2 205;
+		specialeffect2 102;
+		specialeffect2 104;
+		mes "The necklace has been enchanted according to your specifications.";
+		delitem 25088,.@cost; //Fragment of Dream
+		delequip .@part;
+		getitem2 28387,1,1,0,0,0,0,0,.@enchant; //Priest's Necklace
+		close;
+	case 3:
+		S_Cancel;
```
Comment (Pokye) [inline npc/re/merchants/enchan_edda_half_moon.txt:253]: A doubt, why put a function in the case, to call it?
Wouldn't it be better to put the S_Cancel contents; inside this case 3?

**Comment by OptimusM** [comment]
After the instance creation, You can't Enter.

**Comment by Everade** [comment]
> 

Make sure your rAthena is up to date with the required database entries. https://github.com/rathena/rathena/commit/c7ec076a6c1073f4ab9435c5b572cb8368698d16

Otherwise please let us know what errors you ran into.

**Comment by OptimusM** [comment]
@Everade thabk you for your answe.
I have all resources for instance_db quest_db and so on, even if i use an external NPC just for entrance, i have no problems entering to the instance.

https://user-images.githubusercontent.com/40366993/193433388-d20dab6b-914c-4239-b15d-5de94b39d072.mp4

**Comment by Everade** [comment]
remove the quest id 14586 from your quest log and retry.
it's an instance cooldown like most other instances have.

**Comment by idk-whoami** [comment]
> @Everade thank you for your quick answer. I have all resources for instance_db quest_db and so on, even if i use an external NPC just for entrance, i have no problems entering to the instance.
> 
> **Edit** I am testing this on a clean and latest rA, no console errors. https://user-images.githubusercontent.com/40366993/193433388-d20dab6b-914c-4239-b15d-5de94b39d072.mp4



> remove the quest id 14586 from your quest log and retry. it's an instance cooldown like most other instances have.

missing   break;

Capture
```
			mes "Hello, I am a pathway to dreams. How can I help you?";
			next;
			switch ( select("Let me enter your dreams.:Cancel") ) {
			case 1:
				break;
			case 2:
				mes "[Dream Path]";
				mes "Okay, I got it. Please come back again if you change your mind.";
				close;
			}
		} else {
			mes "[Dream Path]";
			mes "Only party leaders may enter.";
			close;
		}
+		break;
	case 0:
	case 1:
		mes "[Dream Path]";
```

**Comment by Everade** [comment]
Ready for merge.

**Review by aleos89 — APPROVED** [review]
Minor fixes!

**Comment by Everade** [comment]
Looks great.

**Pull request merged**